src/AwsInspector/Ssh/Connection.php

```diff
@@ -146,7 +146,16 @@ public function connect()
      */
     public function tunnel($configuration)
     {
-        list($localPort, $remoteHost, $remotePort) = explode(':', $configuration);
+        if (count(explode(':', $configuration)) == 4) {
```

Comment:
I see the explode function 3 times, independently of the condition the same explode will always be executed twice. You can do that once and assign the result in a variable ;-)